### ISSUE TITLE
Rename lr to learning_rate in create_model function

### ISSUE THREAD

**Issue by DOMFIRMUS** [PMEAL/porespy#993]
The create_model function in the src/porespy/networks/_size_factors.py file currently uses lr as a parameter for learning rate, which can be unclear or inconsistent. For improved readability and consistency, we should rename the lr parameter to learning_rate.

Before

![image](https://github.com/user-attachments/assets/7d9cf7ef-a2ee-48f7-9c40-49696e7ed893)

After

![image](https://github.com/user-attachments/assets/d545251b-da54-46c2-bac8-0bc402be2f9a)

**Comment by ma-sadeghi** [comment]
Fixed in be61004b97169003c308a54327b910cd57ee0628
